include/ck/tensor_operation/gpu/element/element_wise_operation.hpp

```diff
@@ -114,29 +114,63 @@ struct AddHardswishAdd
 // E = FastGelu(C + D0 + D1)
 struct AddAddFastGelu
 {
+    // Fast GeLU
+    // https://paperswithcode.com/method/gelu
+    // y = 0.5*x*(1+tanh(sqrt(2/pi)*(x+0.044715*x^3)))
+    __host__ __device__ static constexpr float GetFastGeLU(float x)
+    {
+        const float u   = 2.f * x * (0.035677f * x * x + 0.797885f);
+        const float emu = exp(-u);
+        const float cdf = 0.5f + 0.5f * (2.f / (1.f + emu) - 1.f);
+        return x * cdf;
+    }
+
     template <typename E, typename C, typename D0, typename D1>
-    __host__ __device__ void operator()(E&, const C&, const D0&, const D1&) const;
+    __host__ __device__ constexpr void operator()(E&, const C&, const D0&, const D1&) const
+    {
+        static_assert(
```

Comment:
How about this style?
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/39209849d5e8869976bbf6115863625200617d7e/include/ck/tensor_operation/gpu/element/unary_element_wise_operation.hpp#L149-L157

Reply:
There are slightly differences between specializations. But I think I can do some changes to unify the type-conversion codes in order to share same implementation. So that we can put the guard in front of this shared code.